tests/unit/test_cli.py

```
@@ -1261,3 +1261,184 @@ def test_sync_warns_on_autodiscovered_toml_syntax_error(self, monkeypatch, tmp_p
+                "--env",
+                "soak",
+                "-p",
+                "aws",
+            ],
+        )
+
+        assert result.exit_code == 0
+        assert "soak-machine" in pushed_secrets
+        assert pushed_secrets["soak-machine"] == "DOTENV_PRIVATE_KEY_SOAK=my-secret-key-value"
+
+    def test_vault_push_direct_mode(self, monkeypatch, tmp_path: Path):
```

Ruff (0.14.10)

Unused method argument: `tmp_path`

(ARG002)